kyuubi-server/src/test/scala/org/apache/kyuubi/server/api/v1/OperationsResourceSuite.scala

```diff
@@ -48,15 +48,22 @@ class OperationsResourceSuite extends KyuubiFunSuite with RestFrontendTestHelper
   }
 
   test("apply an action for an operation") {
-    val opHandleStr = getOpHandleStr(OperationType.EXECUTE_STATEMENT)
+    var opHandleStr = ""
+    eventually(Timeout(10.seconds)) {
```

Comment:
any better way rather than repeating?

Reply:
Set state from `INITIALIZED` to `RUNNING` in `be` and keep this state . 
Please review it again when free, thx.